chartist-plugin-legend.js

```diff
@@ -140,74 +92,41 @@
                   options.position.insertBefore(legendElement, null);
                }
             });
+            chart.on('data', function (data) {
+              // empty when ready.
+              $(legendElement).empty();
```

Comment:
You're introducing a jQuery dependency here, could you use another way (I think `legendElement.innerHTML = ''` might work)?